src/main/java/com/mitsuki/jmatrix/util/Options.java

```diff
@@ -470,87 +470,99 @@ public static boolean writeToFile(String filePath, String[ ] contents) {
     ///// --------------------- /////
 
     /**
-    * Raises an error or exception and prints the stack trace of the
-    * input exception object and then exits the program (default exit code: 1).<br>
-    *
-    * @param ex  the {@code Exception} object to be printed.
-    *
-    * @since 1.0.0
-    * @see   #raiseError(Exception, int)
-    * @see   #raiseErrorMsg(Exception)
-    * @see   java.lang.Exception
-    * @see   java.lang.Exception#printStackTrace
-    */
-    public static final void raiseError(final Exception ex) {
-        ex.printStackTrace();
+     * Raises an error or exception and prints the stack traces of the
+     * input {@link Throwable} object and then exits the program (default exit code: 1).
+     *
+     * @param cause  the {@link Throwable} object.
+     *
+     * @since        1.0.0
+     * @see          #raiseError(Exception, int)
+     * @see          #raiseErrorMsg(Exception)
+     * @see          java.lang.Throwable
+     * @see          java.lang.Throwable#printStackTrace
+     */
+    final public static void raiseError(final Throwable cause) {
```

Comment:
Changed the `raiseError` parameter from `Exception ex` to `Throwable cause`, so it can be used by any type of exceptions either `RuntimeException` and `Exception`.

This also changed on these following methods:
  * `raiseError(Exception, int)` -> `raiseError(Throwable, int)`
  * `raiseErrorMsg(Exception)` -> `raiseErrorMsg(Throwable)`
  * `raiseErrorMsg(Exception, int)` -> `raiseErrorMsg(Throwable, int)`